ext/hash/hash_joaat.c

```diff
@@ -44,17 +48,22 @@ PHP_HASH_API void PHP_JOAATUpdate(PHP_JOAAT_CTX *context, const unsigned char *i
 
 PHP_HASH_API void PHP_JOAATFinal(unsigned char digest[4], PHP_JOAAT_CTX * context)
 {
+  uint32_t hval = context->state;
```

Comment:
This looks like a security bugfix for joaat.  If so, it should be a separate (smaller) patch.  This will make it easier to apply to earlier versions of PHP (maybe clear back to 7.2).  It's certainly not part of the serialization proposal.

Reply:
It's not a security fix for joaat, it's a correctness fix. But I will put this into a separate patch for sure.